docs/policies/policies.md

```diff
@@ -0,0 +1,480 @@
+        "issuanceDate": "2021-06-16T18:56:59Z",
+        "expirationDate": "2022-06-16T18:56:59Z",
+        "issuer": "did", // operating environment
+        "credentialSubject": {
+            "type": "MembershipCredential",
+            "holderIdentifier": "bpn",
+            "memberOf": "Catena-X",
+            "status": "Active",
+            "startTime": "2021-06-16T18:56:59Z"
+        }
+    }
+
+| Variable     | Content ([Source](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/architecture/cx-3-2/3.%20Verifiable%20Credentials/Structure%20and%20Formats.md)) |
+| -------- | -------- |
+| "id" | A unique identifier for the credential (UUID) |
+| "@context" | An array of context URLs specifying the context for interpreting the credential |
+| "type" | An array indicating the type of the credential, including "VerifiableCredential" and "MembershipCredentialCX". |
+| "issuanceDate" | The date and time when the credential was issued (in UTC format) |
+| "expirationDate" | The date and time when the credential expires (in UTC format) |
+| "issuer" | The issuer of the credential, identified by a decentralized identifier (did) associated with the operating environment. |
+| "credentialSubject" | An object containing information about the subject of the credential |
+| credentialSubject "type" | The type of the credential subject, which is "MembershipCredential" |
+|  credentialSubject "holderIdentifier" | A unique identifier for the holder of the credential (in this case, "bpn") |
+|  credentialSubject "memberOf" | The organization or group the holder is a member of (in this case, "Catena-X") |
+| credentialSubject "status" | The status of the credential, indicating it is currently "Active"
+| credenitalSubject "startTime" | The date and time when the membership started (in UTC format)
+
+### Specific Catena-X Constraints
+
+>ℹ️ Detailed Information about all possible `Catena-X constraints` can be found here:
+
+- [https://github.com/eclipse-tractusx/ssi-docu](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/architecture/cx-3-2/edc/policy.definitions.md)
+
+### Access & Usage Policies
+
+In Catena-X, a distinction is made between `Access` and `Usage` Policies.
+>ℹ️ Note: `The terms:"usage policy" and "contract policy" are used synonymously!`
+
+- **access policy:** determines whether a particular consumer is offered an asset or not. For example, we may want to restrict certain assets such that only consumers within a particular geography can see them. Consumers outside that geography wouldn't even have them in their catalog.
+- **usage policy or contract policy:** determines the conditions for initiating a contract negotiation for a particular asset. Note that does not automatically guarantee the successful creation of a contract, it merely expresses the eligibility to start the negotiation.
+
+`Access and Usage Policies are not distinguished by a special semantic but by the time of the check.`

```

Comment:
This isn't any kind of code, therefor I would remove the single code fences. If it's an important statement, you can either make it bold or put it into a quote

```suggestion
The Access and Usage Policies are not distinguished by any special semantics, but rather by the time at which they are checked.

```

Reply:
Corrected